uint8 motor_id
int32 current_cmd
